infrastructure/task/scripts/adjust-resource-requests.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+#
+# Lagoon doesn't allow us to set the resource request which results in kubernetes not
+# being able to adjust it's workloades properly.
+# This script allows us to do that, though in an imperfect way.
+# The Script should be obsolote when a vertical pod autoscaler is in place.
+
+# Namespace
+NAMESPACES_RAW=$(kubectl get ns -o jsonpath='{.items[*].metadata.name}')
+NAMESPACES=($NAMESPACES_RAW)
+
+SYSTEM_NAMESPACES=(
+  "calico-system"
+  "cert-manager"
+  "default"
+  "dpl-cms-develop"
+  "grafana"
+  "harbor"
+  "ingress-nginx"
+  "k8up"
+  "kube-node-lease"
+  "kube-public"
+  "kube-system"
+  "kuma-monitoring"
+  "lagoon"
+  "lagoon-core"
+  "loki"
+  "minio"
+  "prometheus"
+  "promtail"
+  "tigera-operator"
+)
+
+echo "Adjusting resource requests in the cluster"
+for NS in "${NAMESPACES[@]}"; do
+  # Skip system namespaces - those we have enough controll over
+  if [[ " ${SYSTEM_NAMESPACES[@]} " =~ " ${NS} " ]]; then
+    continue
+  fi
+  echo "##   $NS    ##"
+  # Pod to be adjusted
+  DEPLOYMENTS=("cli" "nginx" "varnish" "redis")
+
+  # Desired memory request
+  MEMORY_REQUEST="150Mi"
```

Comment:
Consider if it worth noting what the value of this request is based on? Random higher value, experience/observations? Best practice?

Reply:
right now it is just based on a hunch. We need to get the exact numbers from grafana

Reply:
OK. Can we get this info from Grafana now given the data we have available there?